### PR TITLE
[pt-pt] Add minimum required content for European Portuguese localization

### PR DESCRIPTION
**Pull request by ricardoamaro**
# Description

This pull request proposes adding Portuguese from Portugal (pt-pt) localization to the website of Kubernetes per stated Issue #45874. This will increase accessibility for Portuguese users and native speakers. Per community guidelines, it attempts to address and improve the specific differences between Brazilian Portuguese and Portugal's Portuguese, improving inclusivity of visitors and understanding for users from Portugal. I understand that this process ideally requires collaboration with native speakers and potentially expands based on community feedback and will be working to fulfil the requirements of https://kubernetes.io/docs/contribute/localization/

# Tasks

- [x] Home | https://kubernetes.io/docs/home/
- [x] Setup | https://kubernetes.io/docs/setup/
- [ ] Tutorials | https://kubernetes.io/docs/tutorials/kubernetes-basics/, https://kubernetes.io/docs/tutorials/hello-minikube
- [x] Site strings | [All site strings in a new localized TOML file](https://kubernetes.io/docs/contribute/localization/#site-strings-in-i18n)
- [ ] Releases | https://kubernetes.io/releases (All heading and subheading URLs)
- [ ] Content | Translated documents must reside in their own content/**/ subdirectory,

### CODE REVIEW

File: content/pt-pt/docs/home/_index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+approvers:
+- chenopis
```
Comment (sftim) [inline content/pt-pt/docs/home/_index.md:3]:
```suggestion
```
Only include reviewers and approvers if you are confident they speak (European) Portuguese

File: content/pt-pt/docs/setup/_index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+reviewers:
+- brendandburns
+- erictune
+- mikedanese
```
Comment (sftim) [inline content/pt-pt/docs/setup/_index.md:5]:
```suggestion
```
Only include reviewers and approvers if you are confident they speak (European) Portuguese

File: content/pt-pt/docs/setup/_index.md
Comment (sftim) [inline content/pt-pt/docs/setup/_index.md:1]: I would start with the Brazilian Portuguese version of this file and adjust any words that don't look right. I think that will be less effort.

File: content/pt-pt/docs/setup/best-practices/_index.md
Comment (sftim) [inline content/pt-pt/docs/setup/best-practices/_index.md:1]: I would start with the Brazilian Portuguese version of this file and adjust any words that don't look right. I think that will be less effort.

File: content/pt-pt/docs/setup/best-practices/certificates.md
Comment (sftim) [inline content/pt-pt/docs/setup/best-practices/certificates.md:1]: I don't think this is part of minimum required content.

File: content/pt-pt/docs/setup/best-practices/cluster-large.md
Comment (sftim) [inline content/pt-pt/docs/setup/best-practices/cluster-large.md:1]: I don't think this is part of minimum required content.

File: content/pt-pt/docs/setup/best-practices/enforcing-pod-security-standards.md
Comment (sftim) [inline content/pt-pt/docs/setup/best-practices/enforcing-pod-security-standards.md:1]: I don't think this is part of minimum required content.

File: content/pt-pt/docs/setup/best-practices/multiple-zones.md
Comment (sftim) [inline content/pt-pt/docs/setup/best-practices/multiple-zones.md:1]: I don't think this is part of minimum required content.

File: content/pt-pt/docs/setup/best-practices/node-conformance.md
Comment (sftim) [inline content/pt-pt/docs/setup/best-practices/node-conformance.md:1]: I don't think this is part of minimum required content.

File: content/pt-pt/docs/setup/learning-environment/_index.md
Comment (sftim) [inline content/pt-pt/docs/setup/learning-environment/_index.md:1]: I would start with the Brazilian Portuguese version of this file and adjust any words that don't look right. I think that will be less effort.

File: data/i18n/pt-pt/OWNERS [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+# Teams and members are visible at https://github.com/orgs/kubernetes/teams.
+
+reviewers:
+- sig-docs-pt-reviews
+
+approvers:
+- sig-docs-pt-pt-owners
+
+labels:
+- language/pt-pt
```
Comment (sftim) [inline data/i18n/pt-pt/OWNERS:13]:
```suggestion
- language/pt
```

File: data/i18n/pt-pt/OWNERS [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+# Upstream (European Portuguese) localized strings.
```
Comment (sftim) [inline data/i18n/pt-pt/OWNERS:3]:
```suggestion
# Localized strings for Portuguese (Portugal).
# Teams and members are visible at https://github.com/orgs/kubernetes/teams.
```

File: content/pt-pt/docs/tutorials/kubernetes-basics/update/_index.md
Comment (sftim) [inline content/pt-pt/docs/tutorials/kubernetes-basics/update/_index.md:1]: I would start with the Brazilian Portuguese version of this file and adjust any words that don't look right. I think that will be less effort.

File: content/pt-pt/docs/tutorials/kubernetes-basics/scale/_index.md
Comment (sftim) [inline content/pt-pt/docs/tutorials/kubernetes-basics/scale/_index.md:1]: I would start with the Brazilian Portuguese version of this file and adjust any words that don't look right. I think that will be less effort.

File: content/pt-pt/docs/tutorials/kubernetes-basics/expose/_index.md
Comment (sftim) [inline content/pt-pt/docs/tutorials/kubernetes-basics/expose/_index.md:1]: I would start with the Brazilian Portuguese version of this file and adjust any words that don't look right. I think that will be less effort.

File: content/pt-pt/docs/tutorials/kubernetes-basics/explore/_index.md
Comment (sftim) [inline content/pt-pt/docs/tutorials/kubernetes-basics/explore/_index.md:1]: I would start with the Brazilian Portuguese version of this file and adjust any words that don't look right. I think that will be less effort.

File: content/pt-pt/docs/tutorials/kubernetes-basics/deploy-app/_index.md
Comment (sftim) [inline content/pt-pt/docs/tutorials/kubernetes-basics/deploy-app/_index.md:1]: I would start with the Brazilian Portuguese version of this file and adjust any words that don't look right. I think that will be less effort.

File: content/pt-pt/docs/tutorials/kubernetes-basics/create-cluster/_index.md
Comment (sftim) [inline content/pt-pt/docs/tutorials/kubernetes-basics/create-cluster/_index.md:1]: I would start with the Brazilian Portuguese version of this file and adjust any words that don't look right. I think that will be less effort.

File: content/pt-pt/docs/tutorials/hello-minikube.md
Comment (sftim) [inline content/pt-pt/docs/tutorials/hello-minikube.md:1]: I would start with the Brazilian Portuguese version of this file and adjust any words that don't look right. I think that will be less effort.

File: content/pt-pt/docs/setup/production-environment/container-runtimes.md
Comment (sftim) [inline content/pt-pt/docs/setup/production-environment/container-runtimes.md:1]: I don't think this is part of minimum required content.

File: content/pt-pt/docs/setup/production-environment/tools/kubeadm/control-plane-flags.md
Comment (sftim) [inline content/pt-pt/docs/setup/production-environment/tools/kubeadm/control-plane-flags.md:1]: I don't think this is part of minimum required content.

File: content/pt-pt/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md
Comment (sftim) [inline content/pt-pt/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:1]: I don't think this is part of minimum required content.

File: content/pt-pt/docs/setup/production-environment/tools/kubeadm/ha-topology.md
Comment (sftim) [inline content/pt-pt/docs/setup/production-environment/tools/kubeadm/ha-topology.md:1]: I don't think this is part of minimum required content.

File: content/pt-pt/docs/setup/production-environment/tools/kubeadm/high-availability.md
Comment (sftim) [inline content/pt-pt/docs/setup/production-environment/tools/kubeadm/high-availability.md:1]: I don't think this is part of minimum required content.

File: content/pt-pt/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md
Comment (sftim) [inline content/pt-pt/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:1]: I don't think this is part of minimum required content.

File: content/pt-pt/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md
Comment (sftim) [inline content/pt-pt/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md:1]: I don't think this is part of minimum required content.

File: content/pt-pt/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md
Comment (sftim) [inline content/pt-pt/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md:1]: I don't think this is part of minimum required content.

File: content/pt-pt/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md
Comment (sftim) [inline content/pt-pt/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md:1]: I don't think this is part of minimum required content.

File: content/pt-pt/docs/setup/production-environment/turnkey-solutions.md
Comment (sftim) [inline content/pt-pt/docs/setup/production-environment/turnkey-solutions.md:1]: I don't think this is part of minimum required content.

File: content/pt-pt/docs/setup/production-environment/_index.md
Comment (sftim) [inline content/pt-pt/docs/setup/production-environment/_index.md:1]: I would start with the Brazilian Portuguese version of this file and adjust any words that don't look right. I think that will be less effort.

**Review by sftim — COMMENTED** [review]
Thanks for starting this work.

Here's some feedback on the direction of travel.

**Comment by sftim** [comment]
@ricardoamaro You may want to mark this as draft until you feel it's ready to get a review and merge in

File: data/i18n/pt-pt/OWNERS [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+# Teams and members are visible at https://github.com/orgs/kubernetes/teams.
+
+reviewers:
+- sig-docs-pt-reviews
+
+approvers:
+- sig-docs-pt-pt-owners
```
Comment (sftim) [inline data/i18n/pt-pt/OWNERS:10]: For now, I suggest:
```suggestion
- sig-docs-pt-owners
```
but eventually we may want to split the approvers for Portuguese into two separate teams.

**Comment by ricardoamaro** [comment]
Thank you so much for all your help here @sftim

File: content/pt-pt/docs/home/_index.md
Comment (ricardoamaro) [inline content/pt-pt/docs/home/_index.md:1]: Note to self: must review again the definite articles "o" "a"

File: content/pt-pt/docs/setup/learning-environment/_index.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+weight: 20
+---
+
+<!--
+{{/* Existe um redirecionamento do Netlify desta página para /docs/tasks/tools/ */}}
+{{/* O conteúdo desta página existe apenas para fornecer um stub de navegação */}}
+{{/* e para proteger no caso de esse redirecionamento ser um dia removido. */}}
+
+{{/* Se está a localizar esta página, só precisa de copiar o front matter */}}
+{{/* e adicionar um redirecionamento em "/static/_redirects", para a SUA localização. */}}
```
Comment (sftim) [inline content/pt-pt/docs/setup/learning-environment/_index.md:12]: Make sure to add that redirect. And you can drop everything after line 4 once you've done this.

**Comment by tengqm** [comment]
where are we now?  @ricardoamaro

**Comment by tengqm** [comment]
/close
Closing due to inactivity for more than 30 days.
Feel free to reopen and resume this work.

**Pull request closed without merging**